docs/components/modeler/bpmn/escalation-events/escalation-events.md

```diff
@@ -20,14 +20,13 @@ The example above shows the execution of an escalation event:
 
 In BPMN, an `escalation event` references an `escalation`. Escalations can be referenced by one or more escalation events.
 
-An escalation must define an `escalationCode`. The value of this `escalationCode` is used to determine which catch event
-can catch the thrown escalation.
+An escalation must define an `escalationCode`. The value of this `escalationCode` is used to determine which catch event can catch the thrown escalation.
 
 For throwing escalation events, it is possible to define the `escalationCode` as an `expression`. When the event is reached, the expression is evaluated.
 An escalation with the result of this expression is thrown. If no expression is used the statically defined `escalationCode` is used.
 
-For catching escalation events it is not possible to use an `expression`. The `escalationCode` must always be a static value.
-Alternatively, the `escalationCode` can be left empty. A catch event with an empty `escalationCode` will catch **all** thrown escalations.
+For catching escalation events `escalationCode` must be a static value.
```

Comment:
> must be a static value.

💭 We should probably link such statements to https://docs.camunda.io/docs/next/components/concepts/expressions/#expressions-vs-static-values

Reply:
Great idea!

Reply:
I've tried to incorporate this into https://github.com/camunda/camunda-platform-docs/pull/2471/commits/0cea47e66891bcc1389575cefd12018a2472c087.